### ISSUE TITLE
[BUG] Cannot start 1.1.10-2 image

### ISSUE THREAD

**Issue by kthy** [graphite-project/docker-graphite-statsd#209]
**Describe the bug**
When trying to run the newest image (1.1.10-2), nginx displays 502 Bad Gateway and the logs complain about missing `MIDDLEWARE`.

**To Reproduce**
```bash
docker run -d --name graphite \
    --restart=always \
    -p 80:80 \
    -p 2003-2004:2003-2004 \
    -p 2023-2024:2023-2024 \
    -p 8125:8125/udp \
    -p 8126:8126 \
    graphiteapp/graphite-statsd
```

```text
$ docker logs graphite
Traceback (most recent call last):
  File "/opt/graphite/bin/django-admin.py", line 5, in <module>
    management.execute_from_command_line()
  File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/opt/graphite/lib/python3.9/site-packages/django/core/management/__init__.py", line 325, in execute
    settings.INSTALLED_APPS
  File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 79, in __getattr__
    self._setup(name)
  File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 66, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/graphite/lib/python3.9/site-packages/django/conf/__init__.py", line 157, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/opt/graphite/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/graphite/webapp/graphite/settings.py", line 238, in <module>
    globals().update(import_module(SETTINGS_MODULE).__dict__)
  File "/opt/graphite/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/graphite/webapp/graphite/local_settings.py", line 451, in <module>
    ) + MIDDLEWARE
NameError: name 'MIDDLEWARE' is not defined
ok: run: nginx: (pid 60) 10s
```

**Expected behavior**
The graphite frontend is displayed.

**Environment (please complete the following information):**
 - OS flavor: Ubuntu
 - Image version: 1.1.10-2
 - Custom volumes mounted: None

**Additional context**
Running 1.1.10-1 with the same command works as expected.

**Comment by dbidwell94** [comment]
Can confirm this has just happened to me. I have downgraded back to 1.1.8-2 (what I was previously using) for the time being and it has solved my problem.

**Comment by deniszh** [comment]
Fixed in 1.1.10-3, will be built soon.
